tests/src/Context/DrupalContext.php

```diff
@@ -209,4 +209,15 @@ public function theFollowingContentsUsingForWysiwygFields($text_format, TableNod
     return $nodes;
   }
 
+  /**
+   * Update dates in db table field to be in the past.
+   *
+   * @When I make dates from :field in db table :table to be in past
+   */
+  public function iMakeDatesFromFieldInDbTableToBeInPast($field, $table) {
+    db_update($table)
```

Comment:
This update changes all items on the scheduler's table. Maybe, we can specify a item.